src/app/core/services/netsuite/netsuite-configuration/netsuite-import-settings.service.ts

```
@@ -1,23 +1,93 @@
+      importItems: new FormControl(importSettings?.configuration.import_items ?? false),
+      taxCode: new FormControl(importSettings?.configuration.import_tax_items ?? false),
+      importVendorsAsMerchants: new FormControl(importSettings?.configuration.import_vendors_as_merchants ?? false),
+      importNetsuiteEmployees: new FormControl(importSettings?.configuration.import_netsuite_employees ?? false),
+      defaultTaxCode: new FormControl(importSettings?.general_mappings?.default_tax_code?.id ? findObjectByDestinationId(destinationAttribute, importSettings.general_mappings.default_tax_code.id) : null),
+      searchOption: new FormControl('')
+    });
+  }
+
+  static constructPayload(importSettingsForm: FormGroup): NetsuiteImportSettingPost {
+    const expenseFieldArray = importSettingsForm.getRawValue().expenseFields;
+    const mappingSettings = this.constructMappingSettingPayload(expenseFieldArray);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)